src/debug.c

```diff
@@ -1041,6 +1048,10 @@ void _serverAssertPrintClientInfo(const client *c) {
     serverLog(LL_WARNING, "client->conn = %s", connGetInfo(c->conn, conninfo, sizeof(conninfo)));
     serverLog(LL_WARNING, "client->argc = %d", c->argc);
     for (j = 0; j < c->argc; j++) {
+        if (canLogClientArg(c, j)) {
```

Comment:
Just curious, with which condition, the canLogClientArg() could return false value 0?